app/api/api.py

```diff
@@ -0,0 +1,19 @@
+import json
+from flask import Flask, Response, request, jsonify
+from app.model.order import Order
+
+app = Flask(__name__)
+
+orders = Order()
+
+@app.route('/api/v1/orders', methods=['GET'])
+def get_all_orders():
+    if len(orders.get_all_orders())==0:
+        response = jsonify({"message":"You have not ordered yet"}) 
```

Comment:
u can simpe say `if orders. get_all_orders` .. u dont need to query the `length`